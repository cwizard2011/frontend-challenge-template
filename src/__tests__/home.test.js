/* eslint-env jest */

import { shallow } from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'

import Home from '../pages/home'

describe('With Enzyme', () => {
  it('App shows "Hello world!"', () => {
    const home = shallow(<Home />)
    expect(home.find('p').text()).toEqual('Hello World!')
  })
})

describe('With Snapshot Testing', () => {
  it('App shows "Hello world!"', () => {
    const component = renderer.create(<Home />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
