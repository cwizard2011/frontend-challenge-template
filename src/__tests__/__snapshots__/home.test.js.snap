// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Hello world!" 1`] = `
<p
  className="example"
>
  Hello World!
</p>
`;
